src/applications/vaos/services/appointment/transformers.js

```diff
@@ -438,12 +441,13 @@ function createPatientResourceFromRequest(req) {
  * Builds contained array and populates with video conference info and facility info if available
  *
  * @param {Object} appt  VAR appointment object
- * @returns {Array} Array of contained objects of FHIR appointment containing video conference info
+ * @returns {Array|Object} Returns array of contained objects of FHIR appointment containing video conference info or
+ * returns an object containing patient information.
  */
 function setContained(appt) {
   switch (getAppointmentType(appt)) {
     case APPOINTMENT_TYPES.request: {
-      return [createPatientResourceFromRequest(appt)];
+      return createPatientResourceFromRequest(appt);
```

Comment:
I like that this isn't an array anymore, but I wonder if we could also just put the name as the `name` property of the patient object, instead of having `name` be an object with a `text` property inside.

Reply:
Done.